app/models/statistics/total_savings.rb

```diff
@@ -0,0 +1,27 @@
+class Statistics::TotalSavings
+  def to_s
+    if completed_auctions.count > 0
+      Currency.new(total_starting_price - total_winning_bid_amount).to_s
+    else
+      'n/a'
+    end
+  end
+
+  private
+
+  def total_starting_price
```

Comment:
This works since (a - b) + (c - d)... = (a + c) - (b + d)..., but it might need to be revisited if we did averages/medians or such, right? Or am I a math dolt?